tests/README.md

```diff
@@ -2,25 +2,20 @@
 To ensure the whole stack is working as expected and track bugs, a layer based test architecture is needed on OCP. This plan consists of four layers. The first and second layers mentioned below, would be a part of the future automation testing framework on each OCP (x,y,z) release.
 
 #### L1 First Layer: Host OS and Driver Interfaces 
-This layer's test cases will be executed after the cluster is setup, NFD operator is installed and right before the Intel Device Plugins Operator installation. 
-* Test cases for in-tree and OOT tree drivers
+This layer's test cases will be executed after [KMMO](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/kmmo/README.md), [machine configuration](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/README.md) and [NFD operator](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/nfd/README.md) is installed and right before the [Intel Device Plugins Operator](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/device_plugins/README.md) installation. Refer to [readme](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/tests/l1/README.md).
+* Test cases to check existence of in-tree and OOT tree drivers
 * Any test cases for SELinux and host OS security
-* go-based tests cases for devices on all nodes
-* Dependency- rely on driver teams for any other testing result 
-* Future plan to include OOT driver test cases with KMMO CI/CD pipeline- sanity testing.
-* Currently execute as a job. Plan to find an efficient mechanism to deploy and automate on all nodes.
-
+* go-based tests cases to check for devices on all nodes
+  
 #### L2 Second Layer: Device plugin resource provisioning  
-This layer will be executed after the Intel Device Plugins Operator installation and Custom Resource creation. It includes 
+This layer will be executed after the Intel Device Plugins Operator installation and [device plugins](https://github.com/vbedida79/intel-technology-enabling-for-openshift/blob/main/device_plugins/README.md) creation. Refer to [readme](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/tests/l2/README.md) It includes: 
```

Comment:
L2 test cases should be executed after [Deploy Intel Device Plugins on OpenShift](https://github.com/device_plugins/README.md#deploy-intel-device-plugins-on-openshift)